recipes/blockbuster/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+make
+mkdir $PREFIX/bin
+cp $SRC_DIR/blockbuster.x $PREFIX/bin
```

Comment:
No prob, bin is in again. At least we could get rid of the superfluous
$SRC_DIR :)

On Mon, Mar 21, 2016 at 3:35 PM, Johannes Köster <notifications@github.com>
wrote:

> In recipes/blockbuster/build.sh
> <https://github.com/bioconda/bioconda-recipes/pull/1107#discussion_r56831560>
> :
>
> > @@ -0,0 +1,4 @@
> > +#!/bin/bash
> > +make
> > +mkdir $PREFIX/bin
> > +cp $SRC_DIR/blockbuster.x $PREFIX/bin
>
> @bgruening <https://github.com/bgruening> another issue: if you look at this
> recipe
> <https://github.com/bioconda/bioconda-recipes/blob/14f00b12ccdc27bb42379f2f3bb0d860deb20c1d/recipes/krona/2.5/build.sh>,
> it installs the krona source into $PREFIX/krona. This will actually persist
> in the installed package. If that is intended, we should at least stick to
> the FHS and put it into $PREFIX/share/krona or even $PREFIX/src/krona.
> @dmaticzka <https://github.com/dmaticzka> sorry to hijack your PR, once
> the proper copying to bin is there again, I will be happy to merge!
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/bioconda/bioconda-recipes/pull/1107/files/715c481e64853dc7e6c651bd85dbcc711f0bb6fb#r56831560>
>



-- 
--------------------------------------------------
Daniel Maticzka, Dipl. Inf.
Lehrstuhl für Bioinformatik
Institut für Informatik
Albert-Ludwigs-Universität Freiburg

Tel: (+49) 761 203 97529
Fax: (+49) 761 203 7462
Mail: maticzkd@informatik.uni-freiburg.de
--------------------------------------------------